config/gameConfig.json

```
@@ -555,6 +555,22 @@
 					"valueType" : "BASE_NUMBER"
 				}
 			}
+		},
+
+		"general" :
```

Let's call new section 'interface'. Might add more stuff to it later. But probably only after figuring out multiplayer / save compatibility.

```suggestion
		"interface" :
```